### PR TITLE
Build release images automatically

### PR DESCRIPTION
**Pull request by Jannkar**

### CODE REVIEW

File: .github/workflows/build_release.yaml [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+    runs-on: ubuntu-latest
+
+    permissions:
+      contents: read
+      packages: write
+
+    steps:
+      - uses: actions/checkout@v4
+      - name: Log in to the Container registry
+        uses: docker/login-action@65b78e6e13532edd9afa3aa52ac7964289d1a9c1
```
Comment (jonipol) [inline .github/workflows/build_release.yaml:21]: Should we use one of the latest tagged versions of the login-action? This commit is from 2023 March where the latest tagged v3 is from September.

File: .github/workflows/build_release.yaml [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+        with:
+          registry: ${{ env.REGISTRY }}
+          username: ${{ github.actor }}
+          password: ${{ secrets.GITHUB_TOKEN }}
+      - name: Extract metadata (tags, labels) for Docker
+        id: meta
+        uses: docker/metadata-action@9ec57ed1fcdbf14dcef7dfbe97b2010124a938b7
```
Comment (jonipol) [inline .github/workflows/build_release.yaml:28]: The same version question about this one?
https://github.com/docker/metadata-action/tags

File: .github/workflows/build_release.yaml [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+        with:
+          images: ${{ env.REGISTRY }}/${{ env.IMAGE_NAME }}
+      - name: Build and push Docker image
+        uses: docker/build-push-action@f2a1d5e99d037542a71f64918e516c093c6f3fc4
```
Comment (jonipol) [inline .github/workflows/build_release.yaml:32]: The same version question about this one?
https://github.com/docker/build-push-action/tags

**Review by jonipol — COMMENTED** [review]
Questions of the versions used for the actions. Otherwise, seems ok.

File: .github/workflows/build_release.yaml (hunk 1 above)
Comment (Jannkar) [reply .github/workflows/build_release.yaml]: Using now the latest tag for all the actions

**Review by jonipol — APPROVED** [review]

**Pull request merged**